docs/yaml/spec/v2.json

```diff
@@ -0,0 +1,85 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Read the Docs configuration file",
+  "type": "object",
+  "required": [],
+  "properties": {
+
+    "version": {
+      "description": "The version of the spec to use",
+      "type": "string",
+      "enum": ["1", "2"],
+      "default": "1"
+    },
+
+    "formats": {
+      "description": "Formats of the documentation to be built",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": ["htmlzip", "pdf", "epub", "singlehtmllocalmedia"]
+      },
+      "default": ["htmlzip", "pdf", "epub", "singlehtmllocalmedia"],
+      "uniqueItems": true
+    },
+
+    "requirements_file": {
+      "description": "The path to the requirements file from the root of the project",
```

Comment:
I didn't find any format to `path`, so we need to validate this on our side (we need to validate that the path exists anyway)